Representable/representable
# Block group snapping for key states

**changes**
- on entry page, view census block group lines, and highlight them when drawing a polygon (#279 )
- block group files for key states uploaded to mapbox: MI, VA, IL, NJ, TX, PA (waiting to upload others until we decide how best to move forward here + hear from mapbox)
- drawing a community now highlights all polygons which have overlap with the user polygon, rather than just those entirely contained by the polygon (#284 )
- saving polygons without querying census layers no longer gives errors
- removed references to census blocks (button on map page, stop on shepherdjs tour)
- removed geocoded point when searching (clutter, imo)

**to test**
- go to entry page
- try drawing a community in key states
- play around with tools to try and break it
- try drawing in other states & confirm no errors

**screenshots**
before drawing:
![image](https://user-images.githubusercontent.com/41943646/87155922-24ea3800-c2ff-11ea-8b9e-4f010ab066ca.png)
after:
![image](https://user-images.githubusercontent.com/41943646/87155954-39c6cb80-c2ff-11ea-810e-8c70d8213619.png)
